src/widgets/truenas/component.jsx

```diff
@@ -39,9 +42,19 @@ export default function Component({ service }) {
         <Block label="truenas.alerts" value={t("common.number", { value: alertData.pending })} />
       </Container>
       {enablePools &&
+        scaleDeployment &&
```

Comment:
```suggestion
        widget?.nasType === "scale" &&
```

This whole thing could be an if really, i.e. if enablePools --> scale vs else